slides.md

```diff
@@ -680,6 +680,9 @@ BONUS STEP.
 
 # Next Steps
```

Comment:
Can you add a step here and create a deployment guide for Platformatic Cloud using `platformatic deploy` command?